### PR TITLE
anycable-go 0.6.1 (new formula)

### PR DESCRIPTION
**Pull request by sponomarev**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Migrated from custom tap https://github.com/anycable/homebrew-anycable/pull/1

### CODE REVIEW

**Comment by sponomarev** [comment]
Hey! Is there any chance for this PR to be reviewed and merged?

**Comment by palkan** [comment]
@sponomarev we probably need to upgrade to 0.6.0

**Comment by sponomarev** [comment]
@palkan done!

File: Formula/anycable-go.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/anycable/anycable-go/").install Dir["*"]
+    system "go", "build", "-ldflags", "-s -w -X main.version=#{version}", "-o", "#{bin}/anycable-go", "-v", "github.com/anycable/anycable-go/cmd/anycable-go"
+  end
+
+  test do
+    assert_match /#{version}/, shell_output("#{bin}/anycable-go -v")
```
Comment (igas) [inline Formula/anycable-go.rb]: is there a chance to test more functionality? Start a server for example? You can take a look at [armor](https://github.com/Homebrew/homebrew-core/blob/16b8dd3d8c1ccf8cab179d2b967e269648d960cd/Formula/armor.rb#L29-L40) for example.

Comment (palkan) [reply Formula/anycable-go.rb]: Good idea.
Testing WebSocket server could be tricky; we're going to add http healthchecks (https://github.com/anycable/anycable-go/issues/52), we'll update the formula as soon as it released

Comment (sponomarev) [reply Formula/anycable-go.rb]: @palkan Is it possible to run AnyCable without redis? I don't think that it's a good idea require it explicitly in formula and rely on a default configuration. At least, starting redis automatically is an option and it's stopped right after the installation.

Comment (igas) [reply Formula/anycable-go.rb]: Thanks Martians

Comment (sponomarev) [reply Formula/anycable-go.rb]: Hey @igas! Thanks for the suggestion, it has been implemented. Is there anything else  to   concider?

**Comment by igas** [comment]
Thank you 👽 @sponomarev @palkan!

**Pull request closed without merging**